### DISCUSSION TITLE
Is cache deleted when I turn off my phone?

### DISCUSSION THREAD

**Discussion by Themrpie** [vercel/swr#3035] [Q&A]
I'm thinking on adding swr into my react native expo project, but would like to know if the cache persist if I close the app or turn off my phone. If not then I think I would have to use AsyncStorage.

Hopefully somebody can help me with this.
Thanks in advance

**Comment by pere-seapoint** [comment]
SWR won't persist the cache for you if you close the app or turn off the phone, it's in-memory cache. [This](https://swr.vercel.app/docs/advanced/cache#localstorage-based-persistent-cache) section of their docs suggests a strategy to do so.
